mitodl/micromasters
# update chai and chai-as-promised

#### What are the relevant tickets?

none, this was something left off of https://github.com/mitodl/micromasters/pull/3243 because chai-as-promised was not yet compatible with version 4 of chai. Now it is!

#### What's this PR do?

what it says on the can

#### How should this be manually tested?

if the tests are all passing we're good! The only thing I had to change was that `assert.include` no longer does a deep comparison, you have to use `assert.deepInclude` now if you want that.
